Log Context: Improved "Whats New" text (#67285)
grafana/grafana: docs/sources/whatsnew/whats-new-in-v10-0.md

{{< figure src="/media/docs/grafana/screenshot-grafana-10-0-annotation-filtering.png" max-width="750px" caption="Annotation filtering" >}}

### Redesigned and improved Log Context

Our team has recently made enhancements to Grafana's log context feature, resulting in a more seamless and consistent user experience. With the updated user interface, users can expect the same level of functionality and usability in log context as they would in any other logs panel.

Notably, we've added the following new features that streamline the log context experience:

> 1. Log details with actions including a "copy" button for easily copying lines and an eye icon for displaying only selected labels, allowing users to focus on specific information without leaving the log context panel.

Reviewer: ```suggestion
1. Log details with actions including a **Copy** button to easily copy lines and an eye icon to display only selected labels, allowing users to focus on specific information without leaving the log context panel.
```

> 2. A "Wrap Lines" toggle to automatically wrap long lines of text for easier reading and analysis of log entry context directly in log context.

Reviewer: ```suggestion
2. A **Wrap Lines** toggle to automatically wrap long lines of text for easier reading and analysis of log entry context directly in log context.
```

> 3. An "Open in split view" button to execute the context query for a log entry in a split screen in the Explore view.

Reviewer: ```suggestion
3. An **Open in split view** button to execute the context query for a log entry in a split screen in the Explore view.
```

> 3. To execute the context query for a log entry in a split screen in the Explore view we've added a "Open in split view" button.

Reviewer 2: Given the way the enumeration is described ("the following new  features"), maybe:

```suggestion
3. An "Open in split view" button to execute the context query for a log entry in a split screen in the Explore view.
```

> 4. Only for Loki: A quick filter menu that let's you easily refine the context query by selecting and removing labels.

Reviewer: ```suggestion
4. Only for Loki: A quick filter menu that lets you easily refine the context query by selecting and removing labels.
```
